backend/server.js

```diff
@@ -34,6 +35,17 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.use('/', indexRouter);
 app.use('/users', usersRouter);
 
+require('dotenv').config(); // sensitive information
+
+//connect to mongo cluster 
+const uri = process.env.ATLAS_URI;
+mongoose.connect(uri, { useNewUrlParser: true, useUnifiedTopology: true });
+
+const connection = mongoose.connection;
+connection.once('open', () => {
+    console.log("Connection to Mongo Cluster established!");
```

Comment:
Same as the comment below, we should use error logs to record information such as this. Allow us to reduce the chance of sensitive data being exposed to an attacker.